modules/twinkleblock.js

```diff
@@ -1578,6 +1578,11 @@ Twinkle.block.callback.filtered_block_groups = function twinkleblockCallbackFilt
 				return;
 			}
 
+			// Select rangeblock for non-/64 ranges only
+			if (blockPreset.value === 'rangeblock') {
+				blockPreset.selected = Morebits.ip.get64(relevantUserName) !== relevantUserName;
```

Comment:
Wanted to note this separately, but I think we should do this instead, thus also disabling {{rangeblock}} for anything smaller than a /64 as well (/65, etc.)  It also has the greatly added benefit of *not* selecting {{rangeblock}} for functionally-equivalent /64 ranges (e.g., 2409:4051:4E93:5F27:0:0:0:**1**/64).

```suggestion
				blockPreset.selected = !Morebits.ip.get64(relevantUserName);
```

Reply:
Even easier if doing #1314 first

Reply:
Done and done.